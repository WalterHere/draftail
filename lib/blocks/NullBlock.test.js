import React from 'react';
import { shallow } from 'enzyme';
import NullBlock from '../blocks/NullBlock';

describe('NullBlock', () => {
    it('exists', () => {
        expect(NullBlock).toBeDefined();
    });

    it('basic', () => {
        expect(shallow(<NullBlock />)).toBeDefined();
    });
});
