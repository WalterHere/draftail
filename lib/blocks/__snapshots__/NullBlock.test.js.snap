exports[`NullBlock basic 1`] = `
<div>
  Invalid component
</div>
`;
